src/Time.mo

```diff
@@ -0,0 +1,13 @@
+/// System time
+
+import Prim "mo:prim";
+module {
+    /// Current system time given as nanoseconds since 1970-01-01. The system guarantees that
+    /// * the time, as observed by the canister, is monotonically increasing, even across canister upgrades.
+    /// * within an incovation of one entry point, the time is constant.
+    ///
+    /// The system times of different canisters are unrelated, and calls from one canister to another may appear to travel "backwards in time"
+    ///
+    /// Note: While an implementation will likely try to keep the system time close to the real time, this is not formally guaranteed.
+    public let time : () -> Nat64 = Prim.time;
```

Comment:
Is this the time interface we want?

I’d not call it `Time.time()`, but maybe `Time.now()` or `Time.getCurrentTime()`?

Also, I’d have use `Nat` for the type – the `Nat64` is a bit too low level.

Or is there a better type to use here?

Reply:
`Time.now()` sounds good to me.

Reply:
I also like `Time.now`.

Reply:
“more `time` to discuss”  – nice one :-)

Reply:
> within an incovation of one entry point, the time is constant

This seems like an important invariant (e.g., lets us witness system-level effects, like the boundaries between activations).

Is it ensured by the public spec?  Should we add a link if so?

Reply:
This is actually copied from the public spec: https://docs.dfinity.systems/public/v/0.9.1/#system-api-time